### PR TITLE
print training loss and eval loss during training

### LINKED ISSUE #1351: Print the training set‘ evaluate result.
In the DNN model in tf.estimator, only the evaluation result of the validation set is output after the training is completed. It is recommended to print the training set to evaluate the effect after DNN model training.

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/sql-machine-learning/sqlflow/issues/1351

TODO:

1. also support keras
2. support print metrics also.

### CODE REVIEW

File: scripts/test_units.sh [hunk 1]
```diff
@@ -38,6 +38,6 @@ go install ./...
 
 # -p 1 is necessary since tests in different packages are sharing the same database
 # ref: https://stackoverflow.com/a/23840896
-SQLFLOW_log_level=debug go test -v -p 1 ./...  -covermode=count -coverprofile=coverage.out
+SQLFLOW_log_level=debug go test -v -p 1 -timeout 900s ./...  -covermode=count -coverprofile=coverage.out
```
Comment (Yancey0623) [inline scripts/test_units.sh:42]: Why 900s?

File: python/sqlflow_submitter/tensorflow/train.py [hunk 2]
```diff
@@ -72,16 +101,21 @@ def train(is_keras_model,
...
+          train_max_steps=None,
+          eval_start_delay_secs=0,
+          eval_throttle_secs=0,
+          save_checkpoints_steps=100,
+          log_every_n_iter=10,
           is_pai=False,
           pai_table=""):
-    if verbose > 0:
-        tf.get_logger().setLevel(logging.INFO)
+    # if verbose > 0:
```
Comment (Yancey0623) [inline python/sqlflow_submitter/tensorflow/train.py:112]: Add a TODO comment here?

File: pkg/sql/codegen/tensorflow/codegen.go [hunk 3]
```diff
@@ -37,6 +37,16 @@ range: [1, Infinity]`, attribute.IntLowerBoundChecker(1, true)},
 	"train.verbose": {attribute.Int, `[default=0]
 Show verbose logs when training.
 possible values: 0, 1`, attribute.IntChoicesChecker([]int{0, 1})},
+	"train.max_steps": {attribute.Int, `[default=0]
```
Comment (Yancey0623) [inline pkg/sql/codegen/tensorflow/codegen.go:40]: Maybe we need to enhancement the user guide also.

**Review by Yancey0623 — DISMISSED** [review]
LGTM with some comment, but not block merge.

File: pkg/sql/codegen/tensorflow/codegen.go (hunk 3 above)
Comment (typhoonzero) [reply pkg/sql/codegen/tensorflow/codegen.go:40]: `docgen` currently only generate XGBoost documents, will do that in a single PR.

**Review by Yancey0623 — APPROVED** [review]
LGTM
